libcrux-psq/tests/query.rs

```diff
@@ -14,18 +17,18 @@ fn query() {
     // External setup
     let responder_ecdh_keys = DHKeyPair::new(&mut rng);
 
-    let responder_pq_keys = PQKeyPair::new(&mut rng);
+    let responder_pq_keys = libcrux_ml_kem::mlkem768::rand::generate_key_pair(&mut rng);
```

Comment:
Same comment as for the builder below. We should pick a style for both. Here in the call site we just see the difference as well.
I think either could be fine. Either a generic `key_gen` with an algorithm identifier, or explicitly generating the keys. But I think I like one function for all a little better.